sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusSessionProcessorOptions.cs

```diff
@@ -35,19 +35,19 @@ public int PrefetchCount
         /// <summary>Gets or sets a value that indicates whether the processor
         /// should automatically complete messages after the <see cref="ServiceBusSessionProcessor.ProcessMessageAsync"/>
         /// handler has completed processing. If the message handler triggers an exception,
-        /// the message will not be automatically completed. The default value is true.
+        /// the message will not be automatically completed.
         /// </summary>
         ///
-        /// <value>true to complete the message automatically on successful execution of the event handler;
-        /// otherwise, false.</value>
+        /// <value>true to complete the message automatically on successful execution of the message handler; otherwise, false.
```

Comment:
```suggestion
        /// <value><c>true</c> to complete the message automatically on successful execution of the message handler; otherwise, <c>false</c>.
```